doc/source/releases/24.2_announce.rst

```diff
@@ -1,12 +1,322 @@
 
-:orphan:
+===========================================================
+24.2 Galaxy Release (January 2025)
+===========================================================
+
+.. include:: _header.rst
+
+Highlights
+===========================================================
+
+Feature1
+--------
+
+Feature description.
+
+Feature2
+--------
+
+Feature description.
+
+Feature3
+--------
+
+Feature description.
+
+DRAFT CONTENT
+
+* Add wizard-like export view for workflow invocations
+* Workflow Editor Activity Bar + Undo stack UI
+* Masthead Revision
+* Guide users to collection builders
+* Workflow Invocation view improvements
+* Libraries Modernisation: Directory Dataset Picker
+* Improvements to help terms & tool help.
+* Feature - stdout live reporting
+* Allow OAuth 2.0 user defined file sources (w/Dropbox integration)
+
+Please see the `24.2 user release notes <24.2_announce_user.html>`__ for a summary of new user features.
+
+Get Galaxy
+===========================================================
+
+The code lives at `GitHub <https://github.com/galaxyproject/galaxy>`__ and you should have `Git <https://git-scm.com/>`__ to obtain it.
+
+To get a new Galaxy repository run:
+  .. code-block:: shell
+
+      $ git clone -b release_24.2 https://github.com/galaxyproject/galaxy.git
+
+To update an existing Galaxy repository run:
+  .. code-block:: shell
+
+      $ git fetch origin && git checkout release_24.2 && git pull --ff-only origin release_24.2
+
+See the `community hub <https://galaxyproject.org/develop/source-code/>`__ for additional details on source code locations.
+
+
+Admin Notes
+===========================================================
+
+DRAFT CONTENT
+
+* Add job metrics per invocation
+* Allow OAuth 2.0 user defined file sources (w/Dropbox integration)
+* Support high-availability setups for the interactive tools proxy
```

Comment:
```suggestion
* Support high-availability setups for the interactive tools proxy (`learn how to set up your own on the Galaxy Training Network <https://training.galaxyproject.org/training-material/topics/admin/tutorials/interactive-tools/tutorial.html#high-availability-setup-with-postgressql-optional>`__)
```

Maybe it makes sense to include the link as a means to encourage the use of the feature